platform/android/llapp_main.lua

```diff
@@ -75,9 +75,7 @@ C.setenv("TESSDATA_PREFIX", path.."/koreader/data", 1)
 -- create fake command-line arguments
 -- luacheck: ignore 121
 if android.isDebuggable() then
-    arg = {"-d", file or path}
-else
-    arg = {file or path}
+    arg = {"-d"}
```

Comment:
Please keep (the rest of) things as they are. `file` is needed as it is how we can open files delivered by intents when the app is closed. `"-d"` should only be enabled on debug builds. It can be still overriden by "enable debug logs" on developer options.

Reply:
whoops, forgot about the file. fixed now

Reply:
Thanks!. I think it should work as is, but I will test before merging. If, for whatever reason, arg doesn't like to join nil we can write it:

`args = file and {"-d", file} or {"-d"}`
`args = file and {file} or {}`